cypress-io/cypress
# fix: Select component inconsistently render selection styles in dropdown

- Closes https://github.com/cypress-io/cypress/issues/21768

### User facing changelog

Fix issue for the `Select Component` do not show the current value as selected in the dropdown with the visible checkmark and other styling. 

### Additional details

The change is necessary to improve accessibility by clearly indicating the selected `List Item` from the `Dropdown Menu`.

### Steps to test

1. Run the command `yarn dev`
2. Add a new project that hasn't been configured with  `Cypress Test` before.
3. For every dropdown menu, the selected `List Item` should have a `bg-jade-50` background and `tick` checkIcon.

### How has the user experience changed?

#### Before
https://www.loom.com/share/eac3bfa96d1744eaa14a0adec8d1b357

#### After
https://www.loom.com/share/e5291eba62dd4303a7fcb22783ff465c

### PR Tasks
